tasks/cross-ruby.rb

```diff
@@ -92,78 +125,117 @@ def dlls
         end),
         "libc.so.6",
       ]
+    when DARWIN_PLATFORM_REGEX
+      [
+        "/usr/lib/libSystem.B.dylib",
+        "/usr/lib/liblzma.5.dylib",
+        "/usr/lib/libobjc.A.dylib",
```

Comment:
@larskanis @tenderlove I could use a sanity check on these remaining references. This seems reasonable to me. I'll probably just ship an RC and see if we get feedback, but wanted to highlight that I made a decision with incomplete information here.